java/tools/src/test/java/org/apache/arrow/tools/TestFileRoundtrip.java

```diff
@@ -78,19 +77,17 @@ public void testDiffFolder() throws Exception {
 
   @Test
   public void testNotPreparedInput() throws Exception {
-    File testInFile = testFolder.newFile("testIn.arrow");
-    File testOutFile = testFolder.newFile("testOut.arrow");
+    File testInFile = new File(testFolder, "testIn.arrow");
+    File testOutFile = new File(testFolder, "testOut.arrow");
 
     String[] args = {"-i", testInFile.getAbsolutePath(), "-o", testOutFile.getAbsolutePath()};
-
-    // In JUnit 5, since the file itself is not created, the exception and message will be
-    // different.
     Exception exception =
         assertThrows(
-            InvalidArrowFileException.class,
+            IllegalArgumentException.class,
             () -> {
               new FileRoundtrip(System.err).run(args);
             });
-    assertEquals("file too small: 0", exception.getMessage());
+
+    assertTrue(exception.getMessage().contains("input file not found"));
   }
```

Comment:
@vibhatha This part has changed due to the JUnit update.
I saw this review previously from @lidavidm, so I am explaining it.

- https://github.com/apache/arrow/pull/42115#discussion_r1635839465

I believe all the necessary work has been completed.

Reply:
Thanks. Let me review again.